yaml-tests/src/main/java/com/apple/foundationdb/relational/yamltests/YamlTestExtension.java

```diff
@@ -66,13 +66,23 @@ public class YamlTestExtension implements TestTemplateInvocationContextProvider,
     private List<ExternalServer> servers;
     @Nullable
     private final String clusterFile;
+    private final boolean includeMethodInDescriptions;
 
     public YamlTestExtension() {
-        this.clusterFile = null; // it will get it from the environment
+        this(null, false);
     }
 
-    public YamlTestExtension(@Nullable final String clusterFile) {
+    /**
+     * Create a new extension with some configuration.
+     * @param clusterFile a custom cluster file to use, or {@code null} to inherit it from the environment, namely
+     * {@code FDB_CLUSTER_FILE}.
+     * @param includeMethodInDescriptions Set this to {@code true} if publishing test results to something that cannot
```

Comment:
Why not just set this to `true` always? Does the name look bad when running in an IDE or something?

Reply:
And I did some investigation into that HTML report, and it looks like the title there is hardcoded:
https://github.com/gradle/gradle/blob/1f2f66e0320559f3391cee8f79ba96dccd6cebcc/platforms/software/testing-base/src/main/java/org/gradle/api/internal/tasks/testing/report/PageRenderer.java#L76